integreat_cms/cms/models/languages/language_tree_node.py

```diff
@@ -161,7 +161,7 @@ def get_repr(self) -> str:
             f"<LanguageTreeNode (id: {self.id}{language_str}{parent_str}{region_str})>"
         )
 
-    class Meta:
+    class Meta(AbstractTreeNode.Meta):
```

Comment:
The `Meta` class of models derived from `AbstractTreeNode` have to themselves descend from `AbstractTreeNode.Meta` in order to inherit the constraints.
Maybe we should add something to `AbstractTreeNode` that checks whether this is the case for any class derived from it and print a warning otherwise? Sounds hacky and would have to be executed once on subclass definition, not each time an instance is created :confused: